### PR TITLE
added 4 new tweaks

### LINKED ISSUE #262: Request: automatically re-enable rage autocast after combat
**Is your feature request related to a problem? Please describe.**
After combat and post-rage fatigue I always forget to click rage again so it'll be used next combat

**Describe the solution you'd like**
Clicking or right clicking rage keeps it toggled on in combat forever (except when you can't because of fatigue), until I click it again or I run out of rages.

**Describe alternatives you've considered**
Tabletop Tweaks has a way to set it on forever when you have endless rage mythic feat but I didn't want to install their massive mod just for this one thing

### PR DESCRIPTION
**Pull request by Truinto**
Added tweak: Auto Start Rage When Entering Combat
Added tweak: Mass Loot Shows Everything When Leaving Map
Added tweak: Equipment No Weight
Added tweak: Allow Item Use From Inventory During Combat

this closes #262

Remarks:
- Mass Loot Everything still hides items from living enemies, although the Loot All button still loots them. Still really useful as to not miss relic parts.
- Disabling Equipment No Weight only seems to work for the shared inventory after saving and reloading.

### CODE REVIEW

File: ToyBox/classes/MonkeyPatchin/BagOfPatches/Tweaks.cs [hunk 1]
```diff
@@ -273,6 +291,90 @@ static class Tutorial_IsBanned_Patch {
...
+                    if (loot.InteractionLoot != null)
+                    ;
+                }
+#endif
+                return false;
+            }
+        }
+
+        [HarmonyPatch(typeof(BlueprintItem), nameof(BlueprintItem.Weight), MethodType.Getter)]
+        public static class BlueprintItem_NoWeight_Patch1 {
```
Comment (ArcaneTrixter) [inline ToyBox/classes/MonkeyPatchin/BagOfPatches/Tweaks.cs:335]: Did you consider just doing BlueprintItem with weight 0.00(some-number-of-0s)1? It's just checking if you're less than or equal to (double) 0.0 and then forcing it to use the type instead if so.

File: ToyBox/README.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@ Toy Box is a cute and playful mod with 240+ cheats, tweaks and quality of life i
 # Detailed Description
 
 Now with 260+ Cheats, Tweaks and Quality of Life Improvements
-    Tweaks: 84 (or 125 depending on how you count)
+    Tweaks: 88 (or 129 depending on how you count)
```
Comment (cabarius) [inline ToyBox/README.txt:8]: ❤️‍🔥

**Review by cabarius — APPROVED** [review]
Thank you so much for these excellent additions

**Comment by vmnt** [comment]
Awesome!! Thank you!

**Pull request merged**